### ISSUE TITLE
ece2cmor3  ImportError: No module named cmor

### ISSUE THREAD

**Issue by jmcgove** [PCMDI/cmor#186]
Hi
I built CMOR ok. When I then run ece2cmor.py in ece2mor3 I get 
  File "/data/jmcgovern/p-process-ECEarth/other/ece2cmor3-master/ece2cmorlib.py", line 1, in <module>
    import cmor
ImportError: No module named cmor
Do I need to specify something or other?

Thanks!

**Comment by taylor13** [comment]
Could you please provide some information on why this issue was closed?   Were you able to correct the problem?  How?  This would be helpful for other users.
thanks.

**Comment by jmcgove** [comment]
Hi, yes I closed it recently, as I got CMOR to compile ok now. For the uuid error, I had not specified the full correct path to uuid in ./configure. For the cmor error, this was due to the uuid path not having been set correctly. Is that helpful?
When I run some of the ece2cmor3 scripts in the script folder, I get an error it cannot find certain modules. I suppose this is for issues dealing with ece2cmor3 though.
